### ISSUE TITLE
Is this being used in the discord?

### ISSUE THREAD

**Issue by akohli96** [CS-Career-Hackers/resume-bot#6]

**Comment by karimElmougi** [comment]
No, it isn't. The plan is to have the bot limit who can ask for a resume review to people who actually contribute to the channel. Issue #4 is the main one that needs to be addressed before the bot is ready.

**Comment by akohli96** [comment]
I see. The plan is to keep the bot in python?

**Comment by karimElmougi** [comment]
The other bots are all in Python, and the consensus last time this was brought up was to keep all bots using the same stack.

**Comment by akohli96** [comment]
How do we monitor our bot activites etc? Deployment etc

**Comment by karimElmougi** [comment]
I'll let @nikmanG field that one :P

**Comment by nikgil** [comment]
The admins/mods set them up on the CSCH server. They are monitored from there. I try to keep them in docker containers with a separate one for databases.

Regarding the choice of python - yeah it was considered in best interests to keep them together in one language. Makes it easier to debug cross project.
